opendebates/models.py

```diff
@@ -42,6 +42,8 @@ class SiteMode(CachingMixin, models.Model):
     )
     debate_state = models.CharField(max_length=5, null=True, blank=True)
 
+    inline_css = models.TextField(null=True, blank=True)
```

Comment:
Same comment about `null=True` (unless there's a reason the project needs nullable text/char fields?)